scripts/scil_evaluate_bundles_individual_measures.py

```diff
@@ -73,18 +96,34 @@ def compute_measures(filename_tuple):
     density = compute_tract_counts_map(streamlines, dimensions)
     endpoints_density = get_endpoints_density_map(streamlines, dimensions)
 
+    span_list = list(map(compute_span, list(sft.streamlines)))
+    span = float(np.average(span_list))
+    curl = length_avg / span
+    volume = np.count_nonzero(density) * np.product(voxel_size)
+    diameter = 2 * np.sqrt(volume / (np.pi * length_avg))
+    elon = length_avg / diameter
+
     curvature_list = np.zeros((nbr_streamlines,))
     for i in range(nbr_streamlines):
         curvature_list[i] = mean_curvature(sft.streamlines[i])
 
     return dict(zip(['volume', 'volume_endpoints', 'streamlines_count',
                      'avg_length', 'std_length', 'min_length', 'max_length',
+                     'span', 'curl', 'diameter', 'elongation',
                      'mean_curvature'],
-                    [np.count_nonzero(density)*np.product(voxel_size),
-                     np.count_nonzero(endpoints_density) *
-                     np.product(voxel_size),
-                     nbr_streamlines, length_avg, length_std, length_min,
-                     length_max, float(np.mean(curvature_list))]))
+                    [volume, np.count_nonzero(endpoints_density) *
+                     np.product(voxel_size), nbr_streamlines,
+                     length_avg, length_std, length_min, length_max,
+                     span, curl, diameter, elon,
+                     float(np.mean(curvature_list))]))
+
+
+def compute_span(streamline_cords):
+    xyz = np.asarray(streamline_cords)
+    if xyz.shape[0] < 2:
```

Comment:
What is this if for?
If the streamlines has 1 point?
Just to be sure

Reply:
Yes, if a streamline only has one point, then 0 is returned.